woostarter/patterns/hidden-no-search-results.php

```diff
@@ -4,9 +4,7 @@
  * Slug: woostarter/hidden-no-search-results
  * Inserter: no
  */
-declare( strict_types = 1 );
 ?>
-
-<!-- wp:paragraph {"align":"center"} -->
-    <p class="has-text-align-center"><?php echo esc_html_x( 'No search results.', 'Text explaining that there are no results returned from a search', 'woostarter' ); ?>.</p>
+<!-- wp:paragraph {"align":"left","style":{"elements":{"link":{"color":{"text":"var:preset|color|theme-4"}}}},"textColor":"theme-4"} -->
+<p class="has-text-align-left has-theme-4-color has-text-color has-link-color"><?php echo esc_html_x( 'No results were found using that search', 'Text explaining that there are no results returned from a search', 'woostarter' ); ?>.</p>
```

Comment:
```suggestion
<p class="has-text-align-left has-theme-4-color has-text-color has-link-color"><?php echo esc_html_x( 'No results were found using that search.', 'Text explaining that there are no results returned from a search', 'woostarter' ); ?></p>
```

Tiny nitpick but the full-stop is outside of the translation string - I think it's tidier if it's inside with the rest of the text.